databases/heroku-pgbouncer/README.md

```diff
@@ -0,0 +1,22 @@
+# Heroku PgBouncer
+
+## How to run this locally
+
+### Heroku authentication
+
+A heroku token needs to be set with the environment variable `HEROKU_API_KEY`. You can find it in our internal 1Password E2E vault.
```

Comment:
What is this needed for here?

Reply:
Not needed anymore, cleaned it up in `master`. I started with it as a Heroku project. Residue from that.